test/bintray/upload/1

```diff
@@ -0,0 +1,4 @@
+BINTRAY_USER=test TRAVIS_TAG=v1.2.3 TRAVIS_REPO_SLUG=sociomantic/test beaver bintray upload -N -C test -k test -n lala.deb
+0
+jfrog bt version-create --desc= --vcs-tag=v1.2.3 --released= sociomantic/test/test/v1.2.3
```

Comment:
Is the space after `--released` intentional here?

Reply:
Not sure, I mean, `--released` should be a date, and it's empty. Not sure if it should be empty, I will check.

Reply:
Yeah, this is OKish, but not very clean. The released date is taken from the Git tag date, but in the fabricated tests there is no Git tag, hence it is empty. Not ideal but fixing that could be challenging (we basically should have some face git repo to get the info from).

Reply:
Same goes for `--desc=`.